### ISSUE TITLE
testing 4294: no visible focus ring on arrows back/forth for open answers

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3734]
**URL:** result page of e.g. https://meinberlin-dev.liqd.net/projekte/test-poll-merge-running-poll-with-user-content/
**user:** any
**expected behaviour:** I can see that the buttons are in focus when tabbing through site without mouse
**behaviour:** no focus ring for arrows back/forth
**important screensize:**
**device & browser:** big sur, chrome/ff/safari
**Comment/Question:** Same is true for the timeline. This is not part of the story. Would you do that all at once or should I make a new issue?

Screenshot?

**Comment by phillimorland** [comment]
@CarolingerSeilchenspringer I can't replicate this, it's working on chrome and ff for both timeline and answers, on ff there is no focus ring so cou can't tell it's on the arrowes without screen reader, is that what you mean?

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland I can click without mouse but the focus is missing so I cannot see where I am. I mean these little thingis: 
![Bildschirmfoto 2021-07-15 um 17 13 54](https://user-images.githubusercontent.com/35491681/125812396-9ffac3ba-b4bb-4f07-8470-36653d6b3c65.png) This is a screenshot from chrome and the focus should be right here now.

**Comment by phillimorland** [comment]
cool edited issue
